config/crd/bases/keda.sh_kedacontrollers.yaml

```diff
@@ -37,6 +37,8 @@ spec:
             description: KedaControllerSpec defines the desired state of KedaController
             properties:
               logLevel:
+                description: 'Important: Run "make" to regenerate code after modifying
```

Comment:
Not sure if this is the correct description in here? (Imho there should be the same as it is in the sample

Reply:
And encoder is missing here.